deepinterpolation/cli/schemas.py

```diff
@@ -197,6 +197,31 @@ class GeneratorSchema(argschema.schemas.DefaultSchema):
             the maximal number."
     )
 
+    movie_statistics_sample_size = argschema.fields.Int(
+        required=False,
+        default=-1000,
+        description="Number of frames used to calculate normalization \
+            statistics. This constraint to reduce the computational and \
+            space requirements for this calculation."
+    )
+
+    gpu_cache_full = argschema.fields.Boolean(
+        required=False,
+        default=False,
+        description="Cache full movie onto GPU memory for batch generation \
+        when using OphysGenerator. Enabling this will lead to faster batch \
+        generation, training, and inference. If disabled, only the minimum \
+        chunk of movie required for batch generation will be cached onto the \
+        GPU at once."
+    )
+
+    normalize_cache = argschema.fields.Boolean(
```

Comment:
SO these new parameters only work with OphysGenerator. I would either mention that they only apply to OphysGenerator or make them used across the others.